Swiftfin tvOS/Views/VideoPlayer/VideoPlayer.swift

```diff
@@ -72,6 +72,9 @@ struct VideoPlayer: View {
                 .environment(\.isScrubbing, $isScrubbing)
         }
         .onChange(of: videoPlayerManager.currentProgressHandler.scrubbedProgress) { newValue in
+            guard !newValue.isNaN && !newValue.isInfinite else {
+                return
+            }
```

Comment:
4. Live TV playback seems to end up with a scrubbedProgress value that can be infinite or NaN. This causes a crash.
Added a guard to prevent updating the currentProgressHandler in this case.

Reply:
I've always intended for live tv to have its own overlay, I think now is the time to do that.

Reply:
I found a way of adjusting the LiveTVChannelsCoordinator so that I can still use the same VideoPlayer and Overlay. 

If you'd prefer that a new Overlay that is separate and specific to Live video be used instead, I could do that. But does that also mean a new VideoPlayer and VideoPlayerCoordinator, specific to Live video? Or should the same VideoPlayer be used but with different Overlays in different cases?

Reply:
The more I think about this, the more it makes sense to have a separate overlay for Live video. So I guess the question remains whether a separate VideoPlayer is also necessary or not.

Reply:
Ok, latest commit includes new players and overlays specific to Live video. Just to get started they are pretty much duplicates of the regular Video Player and Overlays, I'm sure they might diverge more as we continue to add onto them. I have some ideas on other features I might want to add to Live overlays, but this will give a usable starting point for Live Video under the newer architecture. 

@LePips, please give this a review when you have an opportunity and let me know if you have any feedback. Thanks!